_posts/2025-04-17-libqtxdg-4.2.0.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Release libqtxdg 4.2.0
+slug: libqtxdg-4-2-0
+date: 2025-04-15 07:10
+promoted: true
+categories: release
+---
+
+The LXQt team is proud to announce the release of libqtxdg 4.2.0.
+The release can be downloaded from [Github](https://github.com/lxqt/libqtxdg/releases).
+
+Changes:
+
+ * Adapted to changes in private headers of Qt 6.9.
+
+<br/>
+The complete list of changes is available in our CHANGELOG file.
```

Comment:
Not sure if intended, but this is different from the others which have:

`A full list of changes is in the CHANGELOG file.`